modules/test/playwright/pages/portal-workflow-kaleo-designer-web/NodePropertiesSidebarPage.ts

```diff
@@ -117,10 +115,11 @@ export class NodePropertiesSidebarPage {
 		await this.deleteNotificationsButton.click();
 	}
 
-	async dragTaskNodeToDiagram() {
-		await this.taskNodeSideBarItem.dragTo(
-			this.diagramViewPage.diagramArea,
-			{targetPosition: {x: 200, y: 200}}
-		);
+	async dragNodeToDiagram(nodeLabel: string) {
+		await this.sideBarNodes
+			.getByText(nodeLabel, {exact: true})
+			.dragTo(this.diagramViewPage.diagramArea, {
+				targetPosition: {x: 200, y: 200},
```

Comment:
Hey @larissacribeiro, could you check and call these functions twice. With these static positions, one node will be on top of the other node. We can parameterize those positions if we find some issue.

Reply:
Yes, @igor-franca, you're right! Considering future uses where we would need two or more nodes, it would break, so I've parameterized the function as you said and already included the sidebar back button as it will also be necessary in those scenarios. Could you take another look? thanks!